xml/System/DateTime.xml

```diff
@@ -4547,7 +4547,7 @@ Numerous examples that call the `DateTime.Parse` method are interspersed through
 
 [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
 
-You can also download a complete set of `DateTime.Parse` examples, which are included in a [.NET Core 2.0 project for C#](~/samples/snippets/csharp/downloads/api/System/DateTime.Parse.zip) and a [.NET Core 2.0 project for Visual Basic](~/samples/snippets/visualbasic/downloads/api/System.DateTime.Parse.zip), from the [dotnet/docs GitHub repository](https://github.com/dotnet/docs).
+You can also download a complete set of `DateTime.Parse` examples, which are included in a [.NET Core 2.0 project for C#](https://github.com/dotnet/docs/blob/master/samples/snippets/csharp/downloads/api/System/DateTime.Parse.zip) and a [.NET Core 2.0 project for Visual Basic](https://github.com/dotnet/docs/blob/master/samples/snippets/visualbasic/downloads/api/System/DateTime.Parse.zip), from the [dotnet/docs GitHub repository](https://github.com/dotnet/docs).
```

Comment:
Do you wanna take them directly to the download instead perhaps?
https://github.com/dotnet/docs/raw/master/samples/snippets/csharp/downloads/api/System/DateTime.Parse.zip